src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoUploadDaoMockTest.java

```diff
@@ -447,6 +469,44 @@ public void getStudyUploadsBadOffsetKey() {
         }
     }
     
+    @Test(expectedExceptions = BadRequestException.class)
+    public void getStudyUploadsMinPageSizeEnforced() {
+        StudyIdentifier studyId = new StudyIdentifierImpl("test-study");
+        DateTime startTime = DateTime.now().minusDays(4);
+        DateTime endTime = DateTime.now();
+        
+        dao.getStudyUploads(studyId, startTime, endTime, -1, null);
+    }
+    
+    @Test(expectedExceptions = BadRequestException.class)
+    public void getStudyUploadsMaxPageSizeEnforced() { 
+        StudyIdentifier studyId = new StudyIdentifierImpl("test-study");
+        DateTime startTime = DateTime.now().minusDays(4);
+        DateTime endTime = DateTime.now();
+        
+        dao.getStudyUploads(studyId, startTime, endTime, 101, null);
+    }
+    
+    @Test
+    public void deleteUploadsForHealthCode() {
```

Comment:
How about an equivalent to deleteByHealthCodeSilentlyFails() (for when there are no uploads for the given healthcode)?

Reply:
Added